analyzers/tests/SonarAnalyzer.UnitTest/ResourceTests/FilesToAnalyze/InvalidValuesFileToAnalyze.txt

```diff
@@ -0,0 +1,4 @@
+﻿foo
+123
+C:\Projects\^^^\$\Controllers\HomeController.vb
```

Comment:
I believe this is perfectly valid path. To get some invalid or unexpected stuff:
First one has invalid directory character in the way `:`
Second one is relative path from another drive (valid path, but unexpected). I'd like to see what `Path.GetFileName` actually does in that case
```suggestion
C:\Projects\Controllers:web.config
C:web.config
```